src/app/screens/Onboard/SetPassword/index.tsx

```diff
@@ -42,28 +42,26 @@ export default function SetPassword() {
 
   return (
     <form onSubmit={handleSubmit}>
-      <div className="relative mt-14 lg:flex lg:space-x-8 bg-white dark:bg-surface-02dp pt-12 pb-4 lg:py-12 px-10">
-        <div className="lg:w-1/2">
+      <div className="max-w-[600px] mx-auto relative mt-14 lg:flex lg:space-x-8 bg-white dark:bg-surface-02dp pt-10 pb-10 px-10">
```

Comment:
Is it possible to use the closest tailwind unit instead of 600px?

Reply:
@stackingsaunter could you give a word if you want strict 600px or if you accept something like what ChatGPT has infomed to me?

```
max-w-xl with a value of 576px (36rem), which is 24px less than 600px
max-w-2xl with a value of 672px (42rem), which is 72px more than 600px
``` 

Other thing we can do is the put a custom value in the tailwind.config.js. But confess that it seems too complex/confusing that something like a fragment of a CSS class definition ("a classibute"?) comes from a specific config of a module library. 

Or we can push this discussion further to a whole decision about the use of tailwind.

Reply:
if @stackingsaunter is OK with the button having 576px instead of 600px that's fine to me, I helped choosing exact 600 just because we were following the Figma spec and the issue description

Reply:
this is regarding the box itself, right? 576px is fine

Reply:
Okay, we now have the tailwind unit for 576px